src/api/init.ts

```diff
@@ -149,6 +150,18 @@ export async function startApiServer(opts: {
     res.redirect(`/extended/v1/status`);
   });
 
+  // in case env is not production and url is not provided
+  if (apiDocumentationUrl) {
```

Comment:
These conditions should be inside a single route registration. Also, if the `/doc` endpoint is not enabled, it should return a 404 with a useful message about docs not being available.